src/extension/_locales/de/messages.json

```
@@ -3,74 +3,74 @@
     "message": "Hinzufügen"
   },
   "admin_version": {
-    "message": "Eine spezifische Version des Admin API verwenden"
+    "message": "Eine bestimmte Admin-API-Version verwenden"
   },
   "admin_version_placeholder": {
     "message": "Admin-Version: x.y.z"
   },
   "advanced": {
-    "message": "Fortgeschritten",
-    "description": "Advanced config mode"
+    "description": "Erweiterter Konfigurationsmodus",
+    "message": "Erweitert"
   },
   "anonymous": {
-    "message": "Anonymer Benutzer"
+    "message": "Anonym"
   },
   "bulk_confirm_empty": {
     "message": "Keine Datei ausgewählt"
   },
   "bulk_confirm_preview_multiple": {
-    "message": "Möchtest du wirklich die Vorschau für $1 Dateien generieren?"
+    "message": "Möchten Sie wirklich die Vorschau für $1 Dateien generieren?"
   },
   "bulk_confirm_preview_single": {
-    "message": "Möchtest du wirklich die Vorschau für 1 Datei generieren?"
+    "message": "Möchten Sie wirklich die Vorschau für diese Datei generieren?"
   },
   "bulk_confirm_publish_multiple": {
-    "message": "Möchtest du wirklich $1 Dateien publizieren?"
+    "message": "Möchten Sie wirklich $1 Dateien veröffentlichen?"
   },
   "bulk_confirm_publish_single": {
-    "message": "Möchtest du wirklich 1 Datei publizieren?"
+    "message": "Möchten Sie diese Datei wirklich veröffentlichen?"
   },
   "bulk_progress_preview_multiple": {
-    "message": "Vorschau für $1 von $2 Dateien generiert..."
+    "message": "Vorschau für $1 von $2 Dateien wird generiert …"
   },
   "bulk_progress_publish_multiple": {
-    "message": "$1 von $2 Dateien publiziert..."
+    "message": "$1 von $2 Dateien veröffentlicht …"
   },
   "bulk_result_preview_multiple_failure": {
-    "message": "Vorschau-Generierung für $1 Dateien fehlgeschlagen:"
+    "message": "Für $1 Dateien konnte keine Vorschau generiert werden:"
   },
   "bulk_result_preview_multiple_success": {
     "message": "Vorschau für $1 Dateien erfolgreich generiert."
   },
   "bulk_result_preview_single_error_invalid_folder_name": {
-    "message": "Nur normalisierte lateinische Kleinbuchstaben, Zahlen und Bindestriche sind in Ordnernamen erlaubt"
+    "message": "In Ordnernamen sind nur normalisierte, kleine lateinische Buchstaben, Ziffern und Bindestriche zulässig"
   },
   "bulk_result_preview_single_error_no_docx": {
-    "message": "Microsoft Word-Dokument, zuerst in Google Docs umwandeln"
+    "message": "Microsoft Word-Dokument, muss zuerst in ein Google-Dokument konvertiert werden"
```

2 other occurrences below are now using `zunächst` instead of `zuerst`. We should stay consistent.
```suggestion
    "message": "Microsoft Word-Dokument, muss zunächst in ein Google-Dokument konvertiert werden"
```